src/deluge/gui/menu_item/filter/hpf_mode.h

```diff
@@ -27,12 +27,12 @@ class HPFMode final : public Selection<kNumHPFModes> {
 public:
 	using Selection::Selection;
 	void readCurrentValue() override {
-		this->setValue(util::to_underlying(soundEditor.currentModControllable->hpfMode) - kNumLPFModes);
+		this->setValue(util::to_underlying(soundEditor.currentModControllable->hpfMode) - kFirstHPFMode);
 	}
 	void writeCurrentValue() override {
-		soundEditor.currentModControllable->hpfMode = static_cast<FilterMode>(this->getValue() + kNumLPFModes);
+		soundEditor.currentModControllable->hpfMode = static_cast<FilterMode>(this->getValue() + kFirstHPFMode);
 	}
-	static_vector<std::string, capacity()> getOptions() override { return {"HPLadder", "HPSV"}; }
+	static_vector<std::string, capacity()> getOptions() override { return {"SV_Band", "SV_Notch", "HPLadder"}; }
```

Comment:
This should return a `string_view` probably, `std::string` will force an alloc + copy (even if it worked correctly which it won't until #354 is merged)

Reply:
Oh, if you rebase on to dfa7101e593498392e81615e653a3174ee767b39 the compiler will probably yell at you about this exact thing.